datanode/partition_raft.go

```diff
@@ -436,6 +436,16 @@ func (s *DataNode) startRaftServer(cfg *config.Config) (err error) {
 
 	s.parseRaftConfig(cfg)
 
+	constCfg := config.ConstConfig{
+		Listen:           s.port,
+		RaftHeartbetPort: s.raftHeartbeat,
+		RaftReplicaPort:  s.raftReplica,
+	}
+	if ok, err1 := config.CheckOrStoreConstCfg(s.raftDir, config.DefaultConstConfigFile, &constCfg); !ok {
```

Comment:
Do not use numbers when naming variables.

Reply:
ok